tests/i18n/dates.py

```diff
@@ -6,21 +6,43 @@
 # or later license. See the LICENSE file for a copy of the license and the
 # AUTHORS file for copyright and authorship information.
 
+import pytest
 import time
 from datetime import datetime
 
+from babel import UnknownLocaleError, core as babel_core
 from babel.dates import format_timedelta
 
-from pootle.i18n.dates import localdate, timesince
+from django.utils.translation import get_language, override, to_locale
+
+from pootle.i18n.dates import timesince
 
 
 def test_local_date_timesince(settings):
+    for language in [get_language(), settings.LANGUAGE_CODE, 'en-us']:
+        try:
+            locale = babel_core.Locale.parse(to_locale(language))
+            break
+        except UnknownLocaleError:
+            continue
```

Comment:
Does this test anything? Can we actually have a test that flags the UnkownLocaleError so we know that we're succeeding in capturing that.